docs/app/documentation/component-docs/date-picker/examples/date-picker-i18n-example.component.ts

```diff
@@ -9,29 +9,59 @@ const localized_values = {
         months: ['Jan', 'Fév', 'Mar', 'Avr', 'Mai', 'Juin', 'Juil', 'Aou', 'Sep', 'Oct', 'Nov', 'Déc'],
         fullMonths: ['Janvier', 'Février', 'Mars', 'Avril', 'Mai', 'Juin', 'Juillet',
             'Aout', 'Septembre', 'Octobre', 'Novembre', 'Décembre']
+    },
+    'de': {
+        weekdays: ['Dienstag', 'Mittwoch', 'Donnerstag', 'Freitag', 'Samstag', 'Sonntag', 'Montag'],
+        months: [' Jän', 'Feb', 'März', 'Apr', 'Mai', 'Juni', 'Juli', 'Aug', 'Sept', 'Okt', 'Nov', 'Dez'],
+        fullMonths: ['Januar', 'Februar', 'März', 'April', 'Mai', 'Juni', 'Juli', 'August', 'September', 'Oktober', 'November', 'Dezember']
+    },
+    'bg': {
+        weekdays: ['неделя', 'понеделник', 'вторник', 'сряда', 'четвъртък', 'петък', 'събота'],
+        months: [' яну', 'фев', 'март', 'апр', 'май', 'юни', 'юли', 'авг', 'септ', 'окт', 'ное', 'дек'],
+        fullMonths: ['януари', 'февруари', 'март', 'април', 'май', 'юни', 'юли', 'август', 'септември', 'октомври', 'ноември', 'декември']
     }
 };
 
+@Injectable({
+    providedIn: 'root',
+})
+export class LanguageService {
+
+    selectedLanguage: string = 'bg';
+
+    setLanguage(newLanguage: string) {
+        this.selectedLanguage = newLanguage;
+    }
+
+    getLanguage() { return this.selectedLanguage; }
+}
+
 @Injectable()
 export class CustomCalendarI18n extends CalendarI18n {
 
     // You could also define a custom service and inject it here
-    language: string = 'fr';
+    //language: string = 'fr';
+    languageService: LanguageService;
+
+    constructor(languageService: LanguageService) {
```

Comment:
If you are using a service you should use it correctly with **dependency injection** to set a good example for users. In my opinion you should remove the service and use the method that was present before with the property. It's simpler, and actually fits pretty much every use case. But I don't really mind which is used as long as it's used correctly and has Angular best practices included.